### ISSUE TITLE
Can't run script from the folder with loaders, that doesn't contain `mot/` directory

### ISSUE THREAD

**Issue by rshym** [xen-troops/rcar_flash#22]
It's quite possible use case, to have separate folder with the loaders (../build_domd/.../images/...) and separate folder with the rcar_flash in different places.
It could be great to go to the folder with the loaders and run the command `<path_to>/rcar_flash.py list-boards` or `<path_to>/rcar_flash.py flash ...`.
But in this case rcar_flash reports error
`rcar_flash.py: error: argument --conf: can't open 'rcar_flash.yaml': [Errno 2] No such file or directory: 'rcar_flash.yaml'`

It could be great, if rcar_flash will look for the `mot/` directory in the local directory, and then in the directory it was started from.

**Comment by rshym** [comment]
Perhaps a better option is to make `rcar_flash` installable, rather than fetching it. In this case, the mentioned issue will be solved automatically.

@lorc, what do you think?

**Comment by lorc** [comment]
Yes, I think this is a good idea. Just put a log entry about which loader is used.

**Comment by rshym** [comment]
It was decided that the issue would be fixed by the https://github.com/xen-troops/rcar_flash/pull/29

**Comment by rshym** [comment]
Closing the issue as the corresponding PR is merged.
